Mathlib/Data/ENat/Basic.lean

```diff
@@ -119,8 +119,8 @@ theorem toNat_top : toNat ⊤ = 0 :=
 /-- Recursor for `ENat` using the preferred forms `⊤` and `↑a`. -/
 @[elab_as_elim]
 def recTopCoe {C : ℕ∞ → Sort*} (h₁ : C ⊤) (h₂ : ∀ a : ℕ, C a) : ∀ n : ℕ∞, C n
-| none => h₁
-| Option.some a => h₂ a
+  | none => h₁
+  | Option.some a => h₂ a
```

Comment:
Are you sure this is the right convention?

Reply:
The vast majority of cases are like that, but your question has made me asking for more opinions: 
https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/style.20for.20proofs.20with.20.60.7C.60.2E

Reply:
seems like two spaces is the consensus